python/lsst/ap/pipe/matchSourceInjected.py

```diff
@@ -129,8 +134,79 @@ def run(self, injectedInitialPVICat, diffIm, associatedDiaSources):
         else:
             fakeCat = injectedInitialPVICat
 
+        self._estimateFakesSNR(fakeCat, diffIm)
+
         return self._processFakes(fakeCat, associatedDiaSources)
 
+    def _estimateFakesSNR(self, injectedCat, diffIm):
+        """Estimate the signal-to-noise ratio of the fakes in the given catalog.
+
+        Parameters
+        ----------
+        injectedCat : `astropy.table.Table`
+            Catalog of synthetic sources to estimate the S/N of. **This table
+            will be modified in place**.
+        diffIm : `lsst.afw.image.Exposure`
+            Difference image where the sources were detected.
+
+        Returns
+        -------
+        None
+        """
+        # Create a schema for the forced measurement task
+        schema = afwTable.SourceTable.makeMinimalSchema()
+        schema.addField("x", "D", "x position in image.", units="pixel")
+        schema.addField("y", "D", "y position in image.", units="pixel")
+        schema.addField("deblend_nChild", "I", "Need for minimal forced phot schema")
+
+        # Create the forced measurement task
+        forcedMeas = ForcedMeasurementTask(schema)
+        # Remove plugins that are not required for this task
+        forcedMeas.plugins.pop('base_SdssShape')
+        forcedMeas.plugins.pop('base_GaussianFlux')
+        forcedMeas.plugins.pop('base_TransformedCentroid')
+        forcedMeas.plugins.pop('base_TransformedShape')
+        # Specify the columns to copy from the input catalog to the output catalog
+        forcedMeas.copyColumns = {"coord_ra": "ra", "coord_dec": "dec"}
```

Comment:
I think it would be better to define the config and set the plugins that should be run, instead of removing the ones that should not be run.

Reply:
Done and tested.